src/inputs/input.stories.html

```diff
@@ -0,0 +1,257 @@
+<article>
+  <h2 class="title">Inputs</h2>
+  <p class="title">
+    There are 3 types of input fields defined in the styleguide:
+  </p>
+  <div class="container">
+    <div>
+      <h3 class="title">Underlined Inputs</h3>
+      <form>
+        <div class="omrs-input-group">
+          <input class="omrs-input-underlined" type="text" required />
+          <label>Name</label>
+          <span class="omrs-input-helper">Helper Text</span>
+          <svg
```

Comment:
Shouldn't we be using [styleguide icons](https://styleguide.openmrs.org/?path=/story/openmrs-styleguide--icons) for this? e.g.

```html
<svg class="omrs-icon" role="img">
  <use xlink:href="#omrs-icon-visibility"></use>
</svg>
```